docs/tutorials/periodic.rst

```diff
@@ -111,8 +111,9 @@ calculate the times of the next three transits with
     >>> # NASA Exoplanet Archive for planet properties
     >>> import astropy.units as u
     >>> from astropy.time import Time
-    >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
-    >>> planet_properties = NasaExoplanetArchive.query_planet('TRAPPIST-1 b', all_columns=True)
+    >>> from astroquery.ipac.nexsci.nasa_exoplanet_archive import NasaExoplanetArchive
```

Comment:
The migration to the latest location of the exoplanet archive within `astroquery.ipac.nexsci.nasa_exoplanet_archive.NasaExoplanetArchive` was causing a test failure.